modules/core/include/opencv2/core/hal/intrin_cpp.hpp

```diff
@@ -263,7 +263,8 @@ Most of these operations return only one value.
 
 ### Other math
 
-- Some frequent operations: @ref v_sqrt, @ref v_invsqrt, @ref v_magnitude, @ref v_sqr_magnitude, @ref v_exp
+- Some frequent operations: @ref v_sqrt, @ref v_invsqrt, @ref v_magnitude, @ref v_sqr_magnitude, @ref v_exp,
+                            @ref v_erf
```

Comment:
/build/precommit_docs/4.x/opencv/modules/core/include/opencv2/core/hal/intrin_cpp.hpp:269: warning: unable to resolve reference to 'v_erf' for \ref command